project-code/app/com/edulify/modules/geolocation/GeolocationPlugin.java

```diff
@@ -1,39 +0,0 @@
-package com.edulify.modules.geolocation;
-
-import com.edulify.modules.geolocation.providers.FreegeoipProvider;
-import play.Application;
-
-public class GeolocationPlugin extends play.Plugin {
```

Comment:
It should be documented with a warning that users cannot use both DI Container + Plugin infrastructure together. Or, if it possible, we should avoid the plugin to start if we already have a DI component.

But, for this PR, you can just add the plugin back and after a merge we can work together to figure out how DI container and plugins will live harmoniously.